src/bicep/examples/zeroTrustWorkbook/README.md

```diff
@@ -0,0 +1,96 @@
+# Zero Trust (TIC3.0) Workbook Example
+
+This example adds an Azure Sentinel: Zero Trust (TIC3.0) Workbook solution to MLZ, provided Sentinel has already been deployed; either through the bicep or [terraform implementation instructions](../sentinel/README.md) in the Operations (T1) resource group.
+
+## What this example does
+
+### Deploys a Zero Trust (TIC3.0) Workbook in Azure Sentinel
+
+Documentation can be found here: [Build and monitor Zero Trust (TIC 3.0) security architectures with Microsoft Sentinel](https://docs.microsoft.com/en-us/security/zero-trust/integrate/sentinel-solution)
+
+### Pre-requisites
+
+1. A MissionLZ deployment with Security Center and Sentinel enabled
```

Comment:
Should this say Microsoft Defender for Cloud instead of "Security Center"? (We just implemented the rename in our code this week.)

Reply:
@brooke-hamilton thank you for bringing that to my attention - I was re-reviewing what I had written as it came to light that "Security Center" was recently replaced by "Microsoft Defender for Cloud" - will make sure that is fixed too :)

Reply:
@lisamurphy-msft thanks for the updates on this branch. If the comments and issues are resolved then please mark them as resolved so we know to take another look.